thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java

```diff
@@ -200,12 +202,14 @@ public void init(List<MergedAnomalyResultDTO> anomalies) {
       AnomalyFeedback feedback = anomaly.getFeedback();
       boolean isLabeledTrueAnomaly = false;
       boolean isLabeledTrueAnomalyNewTrend = false;
-      if (feedback != null && feedback.getFeedbackType() != null && feedback.getFeedbackType()
-          .equals(AnomalyFeedbackType.ANOMALY_NEW_TREND)) {
-        isLabeledTrueAnomalyNewTrend = true;
-      } else if (feedback != null && feedback.getFeedbackType() != null && feedback.getFeedbackType()
-          .equals(AnomalyFeedbackType.ANOMALY)) {
-        isLabeledTrueAnomaly = true;
+      if (feedback != null && feedback.getFeedbackType() != null) {
+        AnomalyClassificationType anomalyClassificationType = ResourceUtils.getClassificationType(feedback.getFeedbackType());
+        if (anomalyClassificationType == AnomalyClassificationType.TRUE_POSITIVE) {
```

Comment:
consider use feedback.getFeedbackType().isAnomaly() ? that one is in the feedback type label class, make it easier to manage the labels.

Reply:
Done.